modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/exec/rel/TableScanNodeExecutionTest.java

```diff
@@ -207,7 +208,17 @@ private static class TestInternalTableImpl extends InternalTableImpl {
                     1,
                     Int2ObjectMaps.singleton(0, mock(RaftGroupService.class)),
                     PART_CNT,
-                    addr -> mock(ClusterNode.class),
+                    new ClusterNodeResolver() {
+                        @Override
+                        public ClusterNode getById(String id) {
+                            return mock(ClusterNode.class);
+                        }
+
+                        @Override
+                        public ClusterNode getByConsistentId(String consistentId) {
+                            return mock(ClusterNode.class);
+                        }
+                    },
```

Comment:
I see too many of similar snippets, lets create a test implementation of ClusterNodeResolver.